### DISCUSSION TITLE
Cursor Trail – so fun yet practical! 🥳 Any config recommendations?

### DISCUSSION THREAD

**Discussion by jasonshanks** [kovidgoyal/kitty#8009] [General]
Just wanted to throw out there, been really enjoying the cursor trail new feature today. What a pleasant surprise 😃

At this point I've just:
`cursor_trail = 1`

Has anyone done anything more adventurous they want to share?

Thanks again!

**Comment by kovidgoyal** [comment]
See https://github.com/kovidgoyal/kitty/pull/7970

for some discussion.

**Reply by Ripax** [reply]
does is work for tcsh ?

**Comment by kyoshiro** [comment]
Yes, cursor trail is totally unnecessary but somewhat cool and eye candy must have feature. I love it! Great work!
My settings:
```
cursor_trail 3
cursor_trail_decay 0.1 0.4
```

Not leaving kitty for a day or two...

**Reply by prathmesh-ka-github** [reply]
What does the cursor_trail_decay do?

**Reply by different-name** [reply]
@prathmesh-ka-github 

> Controls the decay times for the cursor trail effect when the `cursor_trail`
is enabled. This option accepts two positive float values specifying the
fastest and slowest decay times in seconds. The first value corresponds to the
fastest decay time (minimum), and the second value corresponds to the slowest
decay time (maximum). The second value must be equal to or greater than the
first value. Smaller values result in a faster decay of the cursor trail.
Adjust these values to control how quickly the cursor trail fades away.

**Reply by prathmesh-ka-github** [reply]
> @prathmesh-ka-github
> 
> > Controls the decay times for the cursor trail effect when the `cursor_trail`
> > is enabled. This option accepts two positive float values specifying the
> > fastest and slowest decay times in seconds. The first value corresponds to the
> > fastest decay time (minimum), and the second value corresponds to the slowest
> > decay time (maximum). The second value must be equal to or greater than the
> > first value. Smaller values result in a faster decay of the cursor trail.
> > Adjust these values to control how quickly the cursor trail fades away.

Thanks

**Comment by sevetis** [comment]
cool
